src/haven/CharWnd.java

```diff
@@ -672,19 +675,20 @@ public void set(boolean a) {
 	}
```

Comment:
to fix size call here pack(); and add override for contentsz like this:
public Coord contentsz() { return super.contentsz().add(UI.scale(5, 2)); } 
this will add small padding to the size

Reply:
Cool! I'm learning a lot from this 👍